third_party/gopkg.in/yaml.v2/README.md

```diff
@@ -0,0 +1,133 @@
+# YAML support for the Go language
```

Comment:
Should we add an explanation like https://github.com/kubernetes/kubernetes/blob/master/third_party/forked/gonum/graph/README.md?

Reply:
I do not see why not :) I will make the change

Reply:
Should we also include the tests from the original repository? 🤔

Reply:
Good question. I don't see a very good reason if we forked from a release and don't plan to do any further "customization".

Reply:
Also the tests [have a dependency to a package](https://github.com/go-yaml/yaml/blob/f6f7691b1fdeb513f56608cd2c32c51f8194bf51/go.mod#L4) that we do not use right now. I will also vote to not include them since we are not making any further changes.